res/values/attrs.xml

```diff
@@ -14,10 +14,12 @@
 	<string-array name="pref_behaviour_entries">
 		<item>@string/pref_behaviour_entries_keep_file</item>
 		<item>@string/pref_behaviour_entries_move</item>
+		<item>@string/pref_behaviour_entries_copy</item>
 	</string-array>
 
 	<string-array name="pref_behaviour_entryValues">
 		<item>NOTHING</item>
 		<item>MOVE</item>
+		<item>COPY</item>
 	</string-array>
-</resources>
\ No newline at end of file
+</resources>
```

Comment:
To better and easier reviews: please try to avoid such changes as they are not changing real code but show up as changes.